openedx/core/djangoapps/catalog/utils.py

```diff
@@ -130,21 +146,14 @@ def get_programs_with_type(types=None, include_hidden=True):
     if programs:
         program_types = {program_type['name']: program_type for program_type in get_program_types()}
         for program in programs:
-            # This limited type filtering is sufficient for current needs and
-            # helps us avoid additional complexity when caching program data.
-            # However, if the need for additional filtering of programs should
-            # arise, consider using the cache_programs management command to
-            # cache the filtered lists of UUIDs.
-            if types and program['type'] not in types:
```

Comment:
We were blocking mitpe programs from appearing on HMS by using the ProgramType filter. Since we have multitenant Course Discovery, we no longer need this filtering.